packages/site/src/data/react/libraries.ts

```diff
@@ -679,11 +701,31 @@ export const libraries: Library<typeof libraryTags[number]>[] = [
 		gitHubRepo: "pmndrs/zustand",
 		npmPackage: "zustand",
 		href: "https://docs.pmnd.rs/zustand/introduction",
-		image: "https://github.com/pmndrs.png",
+		image: "https://docs.pmnd.rs/zustand-resources/bear.png",
 		tags: ["state management"],
 		description:
 			"A small, fast and scalable bearbones state-management solution using simplified flux principles.",
 	},
+	{
+		name: "Jotai",
+		author: "Poimandres",
+		gitHubRepo: "pmndrs/jotai",
+		npmPackage: "jotai",
+		href: "https://jotai.org/",
+		image: "https://storage.googleapis.com/candycode/jotai/jotai-mascot.png",
+		tags: ["state management"],
+		description: "Primitive and flexible state management for React",
+	},
+	{
+		name: "Valtio",
+		author: "Poimandres",
+		gitHubRepo: "pmndrs/valtio",
+		npmPackage: "valtio",
+		href: "https://github.com/pmndrs/valtio",
+		image: "https://github.com/pmndrs.png",
+		tags: ["state management"],
+		description: "makes proxy-state simple",
```

Comment:
Could you maybe add a little bit more detail here? I feel like "proxy-state" is not a term widely understood enough to make the three-word description sufficient for most visitors.

Reply:
To be honest, I haven't used valtio myself - from what I understand, it it like the "Poimandres version of MobX", so another take might be "modern observables" or something like that, but I'm not really sure if that would be fitting. 

I just took the official headline from their homepage. Maybe @dai-shi could provide us with a more fitting description?

Reply:
How about "Lightweight Proxy-Based State Management for ReactJS and VanillaJS"?

Reply:
sounds good, I've adjusted it accordingly